src/devnet-utils.ts

```diff
@@ -79,4 +87,26 @@ export class DevnetUtils implements Devnet {
             return response.data;
         }, "Request failed. Make sure your network has the /postman endpoint");
     }
+
+    public async increaseTime(seconds: number) {
+        return this.withErrorHandler<IncreaseTimeResponse>(async () => {
+            const response = await axios.post<IncreaseTimeResponse>(
+                `${this.endpoint}/increase_time`,
+                {
+                    time: seconds
+                });
+            return response.data;
+        }, "Request failed. Make sure your network has the /increase_time endpoint");
+    }
+
+    public async setTime(seconds: number) {
```

Comment:
Do you think it would make sense to provide an option to set time in `Date` as well? Or does it make more sense to expect users to do the date manipulation anyway?